queries/bibtex/indents.scm

```diff
@@ -0,0 +1,10 @@
+[
+ (entry)
+] @indent
+
+[
+ "{"
+ "}"
+] @branch
+
+(comment) @ignore
```

Comment:
so you could add yourself as maintainer for bibtex? Or we add an auto-import feature

Reply:
"I would prefer not to" :)

Reply:
(see above: not really knowing what I'm doing)

Reply:
the indents and folds were easy enough, though -- those I can manage :)

Reply:
But on a more serious note, I don't think we need an auto-import here -- the highlights queries need tuning, and the other ones are not from upstream.